test/check-application

```diff
@@ -821,10 +821,12 @@ class TestApplication(testlib.MachineCase):
               .selectImageAndDownload() \
               .expectDownloadErrorForNonExistingTag()
 
+    @testlib.skipImage("Arch Linux does not support checkpoint/restore", "arch")
```

Comment:
This reason is misleading -- the lifecycle operations don't test checkpoint/restore. Sure, the test might be broken for a different reason, but maybe call it something like "TODO: investigate"?

Reply:
Ah, will do! I thought it was related to checkpoint and I need to investigate what the test really does honestly.